srcpkgs/qtile/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'qtile'
+pkgname=qtile
+version=0.13.0
+revision=1
+build_style=python3-module
+make_install_args="--skip-build"
```

Comment:
why?

Reply:
Because it's done in `do_build`, so there is no need for build script to check for that in `do_install`.
Micro-optimization kinda thing.

Reply:
Removed.